heat/core/arithmetics.py

```diff
@@ -197,6 +203,66 @@ def bitwise_xor(t1, t2):
     return operations.__binary_op(torch.Tensor.__xor__, t1, t2)
 
 
+def cumprod(a, axis):
+    if axis is None:
```

Comment:
docs

Reply:
this is the line I meant and additional arguments